### ISSUE TITLE
Ending '@see' Attributes with Periods

### ISSUE THREAD

**Issue by InsertCreativityHere** [zeroc-ice/ice#1085]
Currently, when we use `@see` attributes in Slice doc comments, we don't end them in periods. For example:
```
/// The remote invocation metrics map.
/// @see RemoteMetrics
MetricsMap remotes;
```
The code we generate for Java and C# just copy the `@see ...` directly. This is fine for Java, but causes StyleCop warnings in C# (where it wants these lines to end in a period).

Changing the Slice comments to end in a period fixes the C# warnings, but breaks the JavaDoc generation. When JavaDoc sees a period in an `@see` attribute, it expects a package name, and hence for there to be a name after the period.

Should we leave the Slice files as is, and make the C# compiler auto-generate a terminating period?
Or should we add a period to the Slice comments, and make the Java compiler cut out these trailing periods?

**Comment by InsertCreativityHere** [comment]
I think that once we add validation to doc comments, we should reject `@see` ending with periods, since that's the behavior used by JavaDoc: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#see

And then, any languages that want these lines to end in periods (`C#`), we can just generate periods at the ends.

**Comment by InsertCreativityHere** [comment]
This was fixed (along with a couple other things) in #3198.
